client/plugins/supportedFeaturesPlugin.ts

```diff
@@ -31,30 +34,76 @@ declare module 'vuex/types/index' {
 const supportedFeaturesPlugin: Plugin = ({ $browser }, inject) => {
   const supportedFeatures: SupportedFeatures = {
     pictureInPicture: false,
-    airPlay: false
+    airPlay: false,
+    googleCast: false,
+    playbackRate: false,
+    fullScreen: false
   };
 
-  const video = document.createElement('video');
+  /**
+   * Detects if the current platform supports showing fullscreen videos
+   *
+   * @returns {boolean}
+   */
+  function supportsFullscreen(): boolean {
+    // TVs don't support fullscreen. iOS, when user through the PWA, is already full screen.
+    if ($browser.isTv() || ($browser.isApple() && $browser.isMobile())) {
+      return false;
+    }
 
-  if (
-    // Check non-standard Safari PiP support
-    // @ts-expect-error - Non-standard functions doesn't have typings
-    (typeof video.webkitSupportsPresentationMode === 'function' &&
+    const element = document.documentElement;
+
+    return !!(
+      element.requestFullscreen ||
+      // @ts-expect-error -- Non-standard property
+      element.mozRequestFullScreen ||
+      // @ts-expect-error -- Non-standard property
+      element.webkitRequestFullscreen ||
+      // @ts-expect-error -- Non-standard property
+      element.msRequestFullscreen ||
+      // @ts-expect-error -- Non-standard property
+      document.createElement('video').webkitEnterFullscreen
+    );
+  }
```

Comment:
vue fullscreen already does this for you (check for ``support`` boolean): https://www.npmjs.com/package/vue-fullscreen